pkg/traceql/parse_test.go

```diff
@@ -920,6 +921,16 @@ func TestAttributes(t *testing.T) {
 		{in: "parent.span.foo", expected: NewScopedAttribute(AttributeScopeSpan, true, "foo")},
 		{in: "parent.resource.foo.bar.baz", expected: NewScopedAttribute(AttributeScopeResource, true, "foo.bar.baz")},
 		{in: "parent.span.foo.bar", expected: NewScopedAttribute(AttributeScopeSpan, true, "foo.bar")},
+		{in: `."bar z".foo`, expected: NewAttribute("bar z.foo")},
+		{in: `span."bar z".foo`, expected: NewScopedAttribute(AttributeScopeSpan, false, "bar z.foo")},
+		{in: `."bar z".foo."bar"`, expected: NewAttribute("bar z.foo.bar")},
+		{in: `.foo."bar baz"`, expected: NewAttribute("foo.bar baz")},
+		{in: `.foo."bar baz".bar`, expected: NewAttribute("foo.bar baz.bar")},
+		{in: `.foo."bar \" baz"`, expected: NewAttribute(`foo.bar " baz`)},
+		{in: `.foo."bar \\ baz"`, expected: NewAttribute(`foo.bar \ baz`)},
+		{in: `.foo."bar \\"." baz"`, expected: NewAttribute(`foo.bar \. baz`)},
```

Comment:
is this showing you can have multiple quoted strings:

does this parse:
```
."foo bar"foo"foo bar"
```
as an attribute named:
```
foo barfoofoo bar
```
?

Reply:
yes it does, so basically once it's out of quotation it fallbacks to existing logic where it will parse until it finds a attribute terminal character.

Reply:
no. don't change it.

it seemed odd to me at first, but as i'm thinking about the logic this makes the most sense to me. almost no one will use it this way, but i think it's correct to parse this way.